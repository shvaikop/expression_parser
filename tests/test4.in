6
15.567 * (4 - 3    )
21 / (   4 + -20.0) / 6
(123 + ((5) * (-123.4)) -     5)
(123 + ((5 * (-123.4) -     5)
1 * ( 15 + 4) - 71
1 ) * ( 15 + 4) - 71
