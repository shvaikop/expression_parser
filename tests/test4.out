15.567
-0.21875
-499
invalid
-52
invalid
